specification/web/resource-manager/Microsoft.DomainRegistration/stable/2018-02-01/examples/ListTopLevelDomains.json

```diff
@@ -0,0 +1,105 @@
+{
+    "parameters": {
+        "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+        "api-version": "2018-02-01"
+    },
+    "responses": {
+        "200": {
+            "headers": {},
+            "body": {
+                "value": [
+                    {
+                        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.DomainRegistration/topLevelDomains/com",
+                        "name": "com",
+                        "type": "Microsoft.DomainRegistration/topLevelDomains",
+                        "location": "global",
```

Comment:
This example (and that for `TopLevelDomains_Get`) is failing validation right now since the type here is modeled as a  `ProxyOnlyResource` which doesn't have a `location` field. It this intended or do you think that
a) maybe there is a slight modeling bug or
b) these examples should not have `location` on them?

Reply:
Apart from that, please acknowledge whether all changes I point out in my first review (removed example, property changes, etc.) are on purpose. Note that [this](https://github.com/Azure/azure-rest-api-specs/pull/2802#discussion_r178979032) is a real issue, please fix it 🙂

Reply:
@olydis are you referring to the "ManagedServiceIdentity is now a flag enumeration"?

Reply:
correct - just responded to the comment, depending on how that flag enum is supposed to be sent over the wire, it has to be documented accordingly. 🙂